src/main/java/org/folio/rest/service/PubSubService.java

```diff
@@ -62,6 +67,11 @@ public PubSubService(Map<String, String> okapiHeaders, Vertx vertx) {
     return publishAccountBalanceChangeEvent(account);
   }
 
+  public CompletableFuture<Void> publishLoanRelatedFeeFineClosedEvent(Account feeFine) {
+    return pubSubClient.publishEvent(createEvent(LOAN_RELATED_FEE_FINE_CLOSED,
+      LoanRelatedFeeFineClosedEvent.forFeeFine(feeFine).toJsonString()));
+  }
+
   private CompletableFuture<Boolean> publishEvent(final Event event) {
     return PubSubClientUtils.sendEventMessage(event, connectionParams)
```

Comment:
What is the different between the `FeeFinePubSubClient` and the `PubSubClientUtils`?

Reply:
`PubSubClientUtils` is a class from `mod-pubsub-utils` library, which fails when there are no subscribers for the event. 

`FeeFinePubSubClient` - overrides this behavior and treats this case as a success.